src/mds/MDSMap.cc

```diff
@@ -144,6 +144,7 @@ void MDSMap::dump(Formatter *f) const
   f->dump_int("root", root);
   f->dump_int("session_timeout", session_timeout);
   f->dump_int("session_autoclose", session_autoclose);
+  f->dump_string("min_compat_client", ceph_release_name(min_compat_client));
```

Comment:
It'd be useful to also put `min_compat_client` in the output here as version names aren't comparable. (i.e. we can't do a<b)